.circleci/config.yml

```diff
@@ -28,6 +28,10 @@ version: 2
           [[ "$CIRCLE_PR_NUMBER" == "" ]] && ./FTL-client "${FOLDER}" "${BIN_NAME}.sha1" "${FTL_SECRET}"
           rm ./FTL-client
           ls -lah .
+    - run:
+        name: "Test"
```

Comment:
Can this be put before the upload step?

Reply:
The test will only be run for the `x86_64` binary. I intentionally put it after the Upload step so even if the tests fail, we upload the binary. If we put the tests in front of the Upload step, then all binaries would still be uploaded, but the `x86_64` would either be missing or would stay on a previous version.